backend/app/api/questions.py

```diff
@@ -169,7 +176,7 @@ def _parse_table_info(formatting_json: JSON) -> Tuple[int | None, int | None, Re
                 return None, None, post_failure(HTTPErrorCode.WRONG_ID, 404)
             if 'table_column' not in formatting_json or not isinstance(formatting_json['table_column'], int):
                 return None, None, post_failure(HTTPErrorCode.MISSING_ARGUMENT, 400)
-            table_column = formatting_json['table_column']
+            table_column = formatting_json.get('table_column', None)
```

Comment:
Value of `default` parameter in `dict.get(default=None)` is None by default, you don't need to specify this